### ISSUE TITLE
crash/issues when selecting map in list

### ISSUE THREAD

**Issue by terrioss** [Warcraft-III-Community-Edition/W3CE#18]
attached map crashes / launches debugger when selecting it in map list, the issue is with .w3i file

[20:16:04 ERR ] Exception while parsing war3map.w3i!

https://cdn.discordapp.com/attachments/1027304945243926638/1061000558464479283/Legion_TD_5.0b_Team_OZE.w3x

edit: consindeing those are ancient maps, it would be great if they just didnt crash when selecting them

**Comment by terrioss** [comment]
another map

Fatal error. 0xC0000005
   at W3CE.MapScriptAutoStubber.ReadAllBytes(IntPtr, IntPtr, Int32, Int32)
   
   https://cdn.discordapp.com/attachments/1027304945243926638/1061004722317832344/Werewolf_Transylvania_3v._Good_Version.w3x

**Comment by cipherxof** [comment]
I'll add another map to the list: https://wc3maps.com/api/v1/download/191758

^ That map crashes at `W3CE.MapScriptAutoStubber.ReadAllBytes` once the loading screen is finished, on war3mapMap.blp. However, if I re-import the file in a MPQ editor the crash no longer happens, indicating some weirdness with the MPQ.

**Comment by cipherxof** [comment]
> crashes without message
> 
> https://cdn.discordapp.com/attachments/1027304945243926638/1061006183269404712/CHSV_Tournament_Edition7c_Z.1opt.w3x
> 
> edit: another one https://cdn.discordapp.com/attachments/1027304945243926638/1061018501088817182/AngelArenaAllstars2.0.7.w3x , works in reforged

These are fixed for the next release.

> attached map crashes / launches debugger when selecting it in map list, the issue is with .w3i file
> 
> [20:16:04 ERR ] Exception while parsing war3map.w3i!
> 
> https://cdn.discordapp.com/attachments/1027304945243926638/1061000558464479283/Legion_TD_5.0b_Team_OZE.w3x
> 
> edit: consindeing those are ancient maps, it would be great if they just didnt crash when selecting them

This map also crashes in 1.26 and Reforged. It seems to be related to the war3map.j in some way.

> another map
> 
> Fatal error. 0xC0000005 at W3CE.MapScriptAutoStubber.ReadAllBytes(IntPtr, IntPtr, Int32, Int32)
> 
> https://cdn.discordapp.com/attachments/1027304945243926638/1061004722317832344/Werewolf_Transylvania_3v._Good_Version.w3x

This map no longer crashes when you reconfigure the (attributes) file, which indicates some sort of file corruption.

**Comment by cipherxof** [comment]
The only valid map that's still crashing in this issue is `Werewolf_Transylvania_3v._Good_Version.w3x`, so I'm going to close this for now. 

In the future we can open a new issue for each map, or each type of crash and the maps affected by it.
